video_frame_metadata_registry.src.html

```diff
@@ -0,0 +1,70 @@
+<pre class='metadata'>
+Title: WebCodecs VideoFrame Metadata Registry
+Repository: w3c/webcodecs
+Status: DRY
+Shortname: webcodecs-video-frame-metadata-registry
+Level: none
+Group: mediawg
+ED: https://w3c.github.io/webcodecs/video-frame-metadata-registry.html
+TR: https://www.w3.org/TR/webcodecs-video-frame-metadata-registry/
+Editor: Youenn Fablet, w3cid 96458, Apple Inc., youenn@apple.com
+
+Boilerplate: omit conformance
+
+Abstract: This registry is intended to enumerate the metadata fields that can be attached
+    to {{VideoFrame}} objects via the {{VideoFrameMetadata}} dictionary.
+
+Markup Shorthands:css no, markdown yes, dfn yes
+!Participate: <a href="https://github.com/w3c/webcodecs">Git Repository.</a>
+!Participate: <a href="https://github.com/w3c/webcodecs/issues/new">File an issue.</a>
+!Version History: <a href="https://github.com/w3c/webcodecs/commits">https://github.com/w3c/webcodecs/commits</a>
+</pre>
+
+<pre class='anchors'>
+spec: WEBCODECS; urlPrefix: https://w3c.github.io/webcodecs/#
+    type: dictionary
+        text: VideoFrame; url: dictdef-videoframe
+</pre>
+
+
+Registration Entry Requirements {#registration-entry-requirements}
+==================================================================
+
+A registration entry is a document describing one or several metadata entry.
+A registrating entry has the following requirements:
+1. Each metadata entry is defined as a separate {{VideoFrameMetadata}}
+    dictionary member.
+2. Each metadata entry MUST be serializable.
+3. Each metadata entry should have clearly defined semantics.
```

Comment:
The other point here is from @aboba: 

> However, before a registry entry is reviewed by MEDIA WG, there probably needs to be some minimum requirements. For example, that the registry entry refer to a specification that has achieved some level of consensus.

This could apply also to the codecs registry, consider also in a subsequent PR?

Reply:
Done